requirements/dev.txt

```diff
@@ -1,5 +1,4 @@
 -r prod.txt
--r flake8.txt
```

Comment:
What he means is, why did you remove that line? It's needed for our regular tests to run properly so please revert the change to this file :)